tools/xep_tool/xep_tool.escript

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env escript
+%%!  -s inets start
+%% -*- erlang -*-
+%%
+%% Example usage:
+%% add attribute `xep` to relevant module as below:
+%% -xep([{xep, 313}, {version, "0.2"}]).
+%% or
+%% -xep([{xep, 313}, {version, "0.2"},
+%% { url, "http://xmpp.org/extensions/attic/xep-0313-0.2.html"}]).
+%%
+%% you can add also comment to the xep like:
+%% -xep([{xep, 313}, {version, "0.2"},
+%% {comment, "Some example comment, for example: Partial Implemented}]).
+%%
+%% and run:
+%% $> escript xep_tool.escript <PATH_TO_BEAM> <OUTPUT_FILE>
+%% Escript will produce markdown with list of available XEPs.
+
+-export([get_xep/1]).
+-mode(compile).
+-define(HTTP_PREFIX, "http://www.xmpp.org/extensions/xep-").
+-define(HTTP_PREFIX_VER, "http://xmpp.org/extensions/attic/xep-").
+-define(HTTP_SUFIX, ".html").
+-define(XEP, xep).
+-define(VERSION, version).
+-define(URL, url).
+-define(FILE_NAME, "Xep_List.md").
+-define(MAX_LENGTH, 100).
+-type ver() :: string().
+-type xep() :: integer().
+-type url() :: string().
+-type name() :: string().
+-record(xep, {xep :: xep(), name :: name(), url :: url(), version :: ver()}).
+-record(module_xep, {xeps = [], module :: module()}).
+
+-spec get_xep(module()) -> {module(), list({atom(), any()})}.
+get_xep(Module) ->
+    Info = Module:module_info(attributes),
+    Values = [Value || {xep, Value} <- Info],
+    {Module, Values}.
+
+-spec match_zeros(xep()) -> string().
```

Comment:
after changing the returned value to iolist now specs don't match, perhaps also in some other functions